modules/react/_examples/stories/examples/GlobalHeader.tsx

```diff
@@ -45,21 +165,89 @@ const GlobalHeaderItem = createComponent('div')({
 
 const GlobalHeader = createComponent('header')({
   displayName: 'GlobalHeader',
-  Component: (props, ref, Element) => <HeaderWrapper ref={ref} as={Element} {...props} />,
+  Component: (props, ref) => (
+    <header className={styleOverrides.headerWrapper} ref={ref} {...props} />
+  ),
   subComponents: {Item: GlobalHeaderItem},
 });
 
-const HeaderWrapper = styled('header')({
-  display: 'flex',
-  alignItems: 'center',
-  justifyContent: 'space-between',
-  boxSizing: 'border-box',
-  ...type.levels.subtext.large,
-  WebkitFontSmoothing: 'antialiased',
-  MozOsxFontSmoothing: 'grayscale',
-  backgroundColor: colors.frenchVanilla100,
-  ...depth[1],
-  padding: space.xxs,
+const Autocomplete = createComponent('div')({
+  displayName: 'Autocomplete',
+  Component: props => {
+    const [searchText, setSearchText] = React.useState('');
+    const filteredTasks = tasks.filter(i => {
+      if (searchText.trim() === '' || typeof searchText !== 'string') {
+        return true;
+      }
+      return i.toLowerCase().includes(searchText.trim().toLowerCase());
+    });
+
+    function handleChange(e) {
+      setSearchText(e.target.value);
+    }
+
+    return (
+      <Combobox>
+        <InputGroup>
+          <InputGroup.InnerStart cs={styleOverrides.inputGroupInner}>
+            <SystemIcon icon={searchIcon} />
+          </InputGroup.InnerStart>
+          <InputGroup.Input
+            as={AutoCompleteInput}
+            cs={styleOverrides.comboboxInput}
+            onChange={handleChange}
+            value={searchText}
+            {...props}
+          />
+        </InputGroup>
+        <Combobox.Menu.Popper>
+          <Combobox.Menu.Card>
+            {filteredTasks.length === 0 ? (
+              <StyledMenuItem as="span">No Results Found</StyledMenuItem>
+            ) : (
+              filteredTasks.map(i => (
```

Comment:
Although mapping works, we should probably use a render function like in the auto complete example:

```
{model.state.items.length > 0 && (
                <Combobox.Menu.List cs={styleOverrides.comboboxMenuList}>
                  {item => <Combobox.Menu.Item>{item}</Combobox.Menu.Item>}
                </Combobox.Menu.List>
              )}
```

Reply:
I'll need some help with using this model!